features/step_definitions/create_menu_page.rb

```diff
@@ -16,8 +16,17 @@
       path = new_user_registration_path
     when 'restaurants' then
       path = new_restaurant_path
+    when 'create dish' then
```

Comment:
That step def is a bit messy. Leave this for the refactoring of step defs chore?